hackathon_site/hardware/serializers.py

```diff
@@ -1,5 +1,9 @@
+from collections import Counter
+import functools
+import itertools
 from rest_framework import serializers
```

Comment:
Minor, put a blank line between between stdlib imports and module imports (above this line), and between module imports and local imports (below this line)

Reply:
added